nextflow_schema.json

```diff
@@ -52,7 +50,7 @@
                 },
                 "long_reads_assembler_config": {
                     "type": "string",
-                    "description": "Configuration to use flye with. Pick from nano-raw, nano-corr, nano-hq, pacbio-raw, pacbio-corr, pacbio-hifi", 
+                    "description": "Configuration to use flye with. Pick from nano-raw, nano-corr, nano-hq, pacbio-raw, pacbio-corr, pacbio-hifi",
```

Comment:
Is there a way to read these options from an Enum or whatever, somewhere?

Reply:
There is! I didn't notice, I'll change it